BootloaderCorePkg/Include/Library/BootloaderCoreLib.h

```diff
@@ -214,6 +214,7 @@ typedef struct {
   VOID             *PcdDataPtr;
   VOID             *LogBufPtr;
   VOID             *DeviceTable;
+  VOID             *SmbiosPtr;
```

Comment:
Global data structure is more for cross-stage communications.  
For SMBIOS table, since it is built in Stage2 only,  I think the flow can be simpler by doing an allocation in Stage2, and then call SmbiosInit() to update the buffer.   I think it can be handled in the same way as ACPI table.
Putting SMBIOS pointer into Global structure does not seem to be necessary.